.gitignore

```diff
@@ -0,0 +1,24 @@
+*.h5
+*.a
+*.pdf
+bin.block/example_modis_sst/*
+bin.block/example_reynolds_sst/*
+*.grb1
+*.grb2
+
+bin.dp.grib1/example_data/*
+bin.dp.grib2/example_data/*
+dp-p[0-9]*
+bin.revu/example_data/*
+*.swp
+*.docx
+misc/hdf5-1.8.9*
+misc/mpich2-1.4.1.precomp.binaries/*
```

Comment:
Is this repetitive of misc/mpich2-* ?

Reply:
Yes! I have resolved this in my new version.